src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -1 +0,0 @@
-protected net.minecraft.data.recipes.RecipeProvider m_206406_(Lnet/minecraft/tags/TagKey;)Lnet/minecraft/advancements/critereon/InventoryChangeTrigger$TriggerInstance; # has
```

Comment:
Why was this removed?

Reply:
this was removed due to forge has added this AT like 6 months ago